src/utils/format-numbers.js

```diff
@@ -277,21 +303,16 @@ export function formatTimestamp(
   significantDigits: number = 5,
   maxFractionalDigits: number = 3
 ) {
-  // Format in the closest base (hours, minutes, seconds, milliseconds, microseconds, or nanoseconds),
+  // Format in the closest base (days, hours, minutes, seconds, milliseconds, microseconds, or nanoseconds),
   // to avoid cases where times are displayed with too many leading zeroes to be useful.
-  if (time >= 3600 * 1000) {
-    return formatHours(
-      time,
-      significantDigits,
-      Number.isInteger(time / (3600 * 1000)) ? 0 : maxFractionalDigits
-    );
-  }
-  if (time >= 60 * 1000) {
-    return formatMinutes(
-      time,
-      significantDigits,
-      Number.isInteger(time / (60 * 1000)) ? 0 : maxFractionalDigits
-    );
+  if (Math.round(time / 1000) / 60 >= 1) {
+    if (Math.round(time / (60 * 1000)) / 60 >= 1) {
+      if (Math.round(time / (60 * 60 * 1000)) / 24 >= 1) {
```

Comment:
It would be good to add a small comment explaining a few things here:
* the use cases for why you're using Math.round 
* that you're using nested if blocks so that we don't call Math.round repeatidly in the nominal cases